### PR TITLE
update doc for install and angle_class

### PR DESCRIPTION
**Pull request by tink2123**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ For more model downloads (including multiple languages), please refer to [PP-OCR
 - Model training/evaluation
     - [Text Detection](./doc/doc_en/detection_en.md)
     - [Text Recognition](./doc/doc_en/recognition_en.md)
+    - [TEXT ANGLE CLASSIFICATION](./doc/doc_en/angle_class_en.md)
```
Comment (littletomatodonkey) [inline README.md:72]: Direction Classification?

File: doc/doc_ch/installation.md [hunk 2]
```diff
@@ -47,19 +47,16 @@ docker images
...
 ```
 pip3 install --upgrade pip
 
-如果您的机器安装的是CUDA9，请运行以下命令安装
-python3 -m pip install paddlepaddle-gpu==1.7.2.post97 -i https://pypi.tuna.tsinghua.edu.cn/simple
-
-如果您的机器安装的是CUDA10，请运行以下命令安装
-python3 -m pip install paddlepaddle-gpu==1.7.2.post107 -i https://pypi.tuna.tsinghua.edu.cn/simple
+如果您的机器安装的是CUDA9或CUDA10，请运行以下命令安装
+python3 -m pip install paddlepaddle-gpu==1.2.0.beta -i https://pypi.tuna.tsinghua.edu.cn/simple
```
Comment (littletomatodonkey) [inline doc/doc_ch/installation.md:55]: paddlepaddle-gpu==2.0.0b0

File: doc/doc_ch/installation.md [hunk 3]
```diff
@@ -47,19 +47,16 @@ docker images
...
 
 如果您的机器是CPU，请运行以下命令安装
 
-python3 -m pip install paddlepaddle==1.7.2 -i https://pypi.tuna.tsinghua.edu.cn/simple
+python3 -m pip install paddlepaddle==1.2.0.beta -i https://pypi.tuna.tsinghua.edu.cn/simple
```
Comment (littletomatodonkey) [inline doc/doc_ch/installation.md:59]: paddlepaddle==2.0.0b0

File: doc/doc_en/installation_en.md [hunk 4]
```diff
@@ -49,18 +49,15 @@ docker images
...
 ```
 pip3 install --upgrade pip
 
-# If you have cuda9 installed on your machine, please run the following command to install
-python3 -m pip install paddlepaddle-gpu==1.7.2.post97 -i https://pypi.tuna.tsinghua.edu.cn/simple
-
-# If you have cuda10 installed on your machine, please run the following command to install
-python3 -m pip install paddlepaddle-gpu==1.7.2.post107 -i https://pypi.tuna.tsinghua.edu.cn/simple
+# If you have cuda9 or cuda10 installed on your machine, please run the following command to install
+python3 -m pip install paddlepaddle-gpu==2.0.0.beta -i https://pypi.tuna.tsinghua.edu.cn/simple
```
Comment (littletomatodonkey) [inline doc/doc_en/installation_en.md:57]: 同上

File: doc/doc_en/installation_en.md [hunk 5]
```diff
@@ -49,18 +49,15 @@ docker images
...
 
 # If you only have cpu on your machine, please run the following command to install
-python3 -m pip install paddlepaddle==1.7.2 -i https://pypi.tuna.tsinghua.edu.cn/simple
+python3 -m pip install paddlepaddle==2.0.0.beta -i https://pypi.tuna.tsinghua.edu.cn/simple
```
Comment (littletomatodonkey) [inline doc/doc_en/installation_en.md:60]: 同上

**Review by littletomatodonkey — APPROVED** [review]
Excellllent job! LGTM

File: doc/doc_ch/installation.md (hunk 2 above)
Comment (dyning) [reply doc/doc_ch/installation.md]: 源换成官网的源

Comment (tink2123) [reply doc/doc_ch/installation.md]: done

**Pull request merged**